src/pages/ScoreReport.vue

```diff
@@ -883,150 +911,197 @@ const viewOptions = ref([
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Num Correct`] = score.numCorrect;
-      } else if (tasksToDisplayCorrectIncorrectDifference.includes(taskId)) {
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Correct/Incorrect Difference`] =
-          score.correctIncorrectDifference;
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Num Incorrect`] = score.numIncorrect;
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Num Correct`] = score.numCorrect;
-      } else if (tasksToDisplayTotalCorrect.includes(taskId)) {
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Num Correct`] = score.numCorrect;
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Num Attempted`] = score.numAttempted;
-      } else if (rawOnlyTasks.includes(taskId)) {
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Raw`] = score.rawScore;
-      } else {
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Percentile`] = score.percentileString;
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Standard`] = score.standardScore;
-
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Raw`] = score.rawScore;
-        tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Support Level`] = score.supportLevel;
+        tableRow[`${taskName} - Reliability`] = 'Reliable';
       }
 
-      if (score.reliable !== undefined && !score.reliable && score.engagementFlags !== undefined) {
-        const engagementFlags = Object.keys(score.engagementFlags);
-        if (engagementFlags.length > 0) {
-          if (includedValidityFlags[taskId]) {
-            const filteredFlags = Object.keys(score.engagementFlags).filter((flag) =>
-              includedValidityFlags[taskId].includes(flag),
-            );
-            if (filteredFlags.length === 0) {
-              tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Reliability`] = 'Unreliable';
-            } else {
-              const engagementFlagString = 'Unreliable: ' + filteredFlags.map((key) => _lowerCase(key)).join(', ');
-              tableRow[`${tasksDictionary.value[taskId]?.publicName ?? taskId} - Reliability`] = engagementFlagString;
+      // Add progress immediately after reliability if includeProgress is true
+      if (includeProgress) {
+        const progressRow = computedProgressData.value.find(
+          (progress) => progress.userPid === _get(user, 'assessmentPid'),
+        );
+
+        if (progressRow) {
+          scoreReportColumns.value.forEach((column) => {
+            const { field, header: taskName } = column; // Use taskName from the column header
+
+            // Ensure field is defined and is a string before calling startsWith
+            if (field && typeof field === 'string' && field.startsWith('scores')) {
```

Comment:
This can be simplified if I'm not mistaken: 

```suggestion
            if (typeof field === 'string' && field.startsWith('scores')) {
```